Queries/AddedToLiquidationPool.psql

```diff
@@ -0,0 +1,11 @@
+with pass1 as (
+  select sender_address
+  from transactions txs
+  where contract_call_contract_id = 'SP2C2YFP12AJZB4MABJBAJ55XECVS7E4PMMZ89YZR.arkadiko-liquidation-pool-v1-1'
+  and contract_call_function_name = 'stake'
+  and status = 1
```

Comment:
Q: what does status 1 mean?

Reply:
1 for successful, -1 for failed transaction